src/components/Assets/AddAssetComponent.jsx

```diff
@@ -6,6 +6,15 @@ import DropdownComponent from '../common/DropdownComponent';
 import InputFluid from '../common/TextInputComponent';
 import ArtButton from '../common/ButtonComponent';
 
+const numberOfYears = () => (
+  (new Date()).getFullYear() - 2012
+);
+
+const yearOfManufacture = Array(numberOfYears()).fill({}).map((value, index) => ({
```

Comment:
The fact that `value` is not being used is an indication that there's a better way to do this. What are you trying to achieve with `numberOfYears` and `yearOfManufacture` functions in lay man's words? Have a look at [momentjs](https://momentjs.com/docs/) to see if it will simplify things

Reply:
We wanted an array with an object for the years 2013 - the current year. We saw it would be quicker to do it this way and not use a library. I'll delete value and just use index.

Reply:
@sachungo without value the dropdown won't have a key and we get the error 
`Encountered two children with the same key ReactJS`

Reply:
<img width="393" alt="screen shot 2018-08-01 at 3 58 30 pm" src="https://user-images.githubusercontent.com/5074111/43523037-cbc59736-95a3-11e8-83f9-add6c3b1e9e1.png">

Reply:
Ok...and please ensure it still works after the change